### PR TITLE
Add font-inter css files to exports

### PR DESCRIPTION
**Pull request by dohomi**
this PR adds the missing exports for css files of font-inter

### CODE REVIEW

File: packages/font-inter/package.json [hunk 1]
```diff
@@ -14,11 +14,21 @@
...
+    "./css/700.css": "./css/700.css",
+    "./css/800.css": "./css/800.css",
+    "./css/900.css": "./css/900.css",
     ".": {
       "types": "./types/index.d.ts",
       "import": "./dist/esm/index.mjs",
       "require": "./dist/cjs/index.js",
-      "react-native": "./dist/cjs/index.native.js"
+      "react-native": "./dist/cjs/index.native.js",
+      ""
```
Comment (natew) [inline packages/font-inter/package.json:31]: typo?

Comment (dohomi) [reply packages/font-inter/package.json]: fixed the typo

**Comment by dohomi** [comment]
I'm too late to the party.. already merged

**Comment by adyranov** [comment]
@natew font-silkscreen might need to export ./css/400.css as well, it was affected by the same commit as font-inter https://github.com/tamagui/tamagui/commit/4e1b4f5748911d0b4727f4d423b5d8c7c8fa9bc8#diff-e0dddbf5df2cc3f79f612075812cf9c9c29e6e13a64cc5b9ea28e28e2f9f4e4d

**Comment by hirvesh** [comment]
Hey @natew  - trying to use font-silkscreen and the font is not exported.

**Pull request closed without merging**